.github/workflows/ci.yaml

```diff
@@ -119,10 +119,13 @@ jobs:
       registry: ${{ steps.prepenv.outputs.registry }}
       repository: ${{ steps.prepenv.outputs.repository }}
 
-  build-amd64:
-    name: Build amd64 docker image
-    needs: [prepare, tests, linting]
+  build:
+    name: Build images
     runs-on: ubuntu-latest
+    needs: [prepare]
```

Comment:
why is testing and linting not required anymore?

Reply:
whoops